charts/seed-controlplane/charts/etcd/templates/etcd.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: druid.gardener.cloud/v1alpha1
+kind: Etcd
```

Comment:
Currently the hibernation does not scale down the etcd StatefulSets. It actually scales down, but the Etcd resource brings the StatefulSet replicas to 1. Could you please address this?

Reply:
Missed that. Thanks.